env/env_test.cc

```diff
@@ -1271,7 +1401,6 @@ TEST_P(EnvPosixTestWithParam, MultiRead) {
             }
           }
         });
-    ROCKSDB_NAMESPACE::SyncPoint::GetInstance()->EnableProcessing();
```

Comment:
unrelated change?

Reply:
It's a bit weird that i did not change that.